privacyidea/lib/eventhandler/base.py

```diff
@@ -140,6 +141,11 @@ def conditions(cls):
                           "not exist in the userstore anymore."),
                 "value": ("True", "False")
             },
+            CONDITION.TOKEN_IS_EXPIRED: {
+                "type": "str",
+                "desc": _("The token has an expired validity period"),
```

Comment:
Thanks a lot. This is just fine for the PR!
I might add the more detailed thingy later...